deploy/list-connectors.ts

```diff
@@ -0,0 +1,108 @@
+import { glob } from "tinyglobby";
+import fs from "node:fs/promises";
+import { join } from "node:path";
+import Module from 'node:module';
+
+const ROOT_DIR = join(__dirname, "..");
+
+const IGNORE_PACKAGE_LIST = ["api", "backend", "shared"];
+
+// @ts-ignore
+const originalLoad = Module._load;
+
+// mocuk the module loading to prevent errors when a module is not found
+// @ts-ignore
+Module._load = function(path: string) {
+  // mock typia
+  if (path.includes("typia")) {
+    // prevent mocking the typia/lib/transform module
+    if(path.includes("typia/lib/transform")) {
+      return originalLoad.apply(this, arguments);
+    }
+    console.warn(`Warning: Mock typia module`);
+    return {
+      createIs: () => {}
+    };
+  }
+  try {
+    return originalLoad.apply(this, arguments);
+  } catch (error) {
+    // ignore integral package import errors
+    // @ts-ignore
+    if(path.includes("@wrtnlabs")) {
+      console.warn(`Warning: Module "${path}" not found. Using mock instead.`);
+      return {}
+    }
+    // mock typia
+    return originalLoad.apply(this, arguments);
+  }
+};
+
+
+async function main() {
+  const packages = await glob(
+    ["packages/*", ...IGNORE_PACKAGE_LIST.map((name) => `!packages/${name}`)],
+    {
+      onlyDirectories: true,
+      absolute: true,
+    },
+  );
+
+  /** extract package name from package.json */
+  const extractPackageInfo = async (packagePath: string) => {
+    const directory = packagePath.split("/").at(-2);
+    if (!directory) {
+      throw new Error("Invalid package path");
+    }
+
+    const { ENV_LIST = [] }: { ENV_LIST: string[] } = await import(
+      `../packages/${directory}/src/index`
+    );
```

Comment:
just use `join(ROOT_DIR,'packages',directory,'src/index')` is fine. we don't need to split it. just use absolute path for importing